docs/import/color_function_util.js

```diff
@@ -344,6 +336,9 @@ function switchSchema(type) {
 function displaySchema(type) {
   const colorFunction = getColorFunction();
   $('.color-type-div').hide();
+  const byPropertyDiv = $('#by-property');
+  const continuousDiv = $('#continuous');
+  const discreteDiv = $('#discrete');
   switch (type) {
     case ColorStyle.SINGLE:
       $('#single-color-picker').val(colorFunction['color']);
```

Comment:
In the case that no color has been chosen yet, this has weird behavior. The select shows no text, and the td is clear: 
![Screen Shot 2019-12-27 at 12 05 21 PM](https://user-images.githubusercontent.com/10134896/71525762-35761c00-28a1-11ea-8447-d230a1c2654b.png)
But clicking on the select shows that red is selected: 
![Screen Shot 2019-12-27 at 12 05 33 PM](https://user-images.githubusercontent.com/10134896/71525776-4888ec00-28a1-11ea-8087-28bc77c30ab4.png)
But if you don't actually click on anything, it remains clear.

I don't think this is new in this PR, but since you're dealing with unset values, what do you think of generating a default color, perhaps based on the layer name so it's deterministic but not always the same, at layer creation? That way there's a sensible default, as opposed to having this confusing "non-color" state that's only reachable right after layer creation. Could also add a TODO if you think it's a reasonable idea but want to leave for later.